src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_update.py

```diff
@@ -22,6 +22,9 @@ class Update(AAZCommand):
 
     :example: Update custom properties
         az apic api update -g contoso-resources -n contoso --api-id echo-api --custom-properties '{\"public-facing\":true}'
+
+    :example: Update single custom metadata
+        az az apic api update -g contoso-resources -n contoso --api-id echo-api --set customProperties.internal=false
```

Comment:
Please remove the duplicate `az` in the example. `az` is added by default so you don't need to include it when authoring samples in aaz-dev

Reply:
removed